src/Nri/Ui/Select/V3.elm

```diff
@@ -0,0 +1,106 @@
+module Nri.Ui.Select.V3
+    exposing
+        ( Config
+        , customView
+        , view
+        )
+
+{-| Build a select input.
+
+
+## Upgrading to V3
+
+  - Remove the old styles. V3 uses compile-at-runtime elm-css!
+
+
+# Configure
+
+@docs Config
+
+
+# Render
+
+@docs view, customView
+
+-}
+
+import Css
+import Css.Foreign
+import Dict
+import Html.Styled as Html exposing (..)
+import Html.Styled.Attributes as Attributes exposing (..)
+import Html.Styled.Events exposing (..)
+import Json.Decode exposing (Decoder, andThen, succeed)
+import Nri.Ui
+import Nri.Ui.Colors.V1
+import Nri.Ui.Styles.V1
+import Nri.Ui.Util
+
+
+{-| Select-specific Choice.Config
+-}
+type alias Config a =
+    { choices : List { label : String, value : a }
+    , current : a
+    }
+
+
+{-| TODO: move this to View.Extra
+-}
+niceId : String -> a -> String
+niceId prefix x =
+    prefix ++ "-" ++ Nri.Ui.Util.dashify (Nri.Ui.Util.removePunctuation (toString x))
+
+
+{-| A normal select dropdown
+-}
+view : Config a -> Html a
+view config =
+    customView [] config
+
+
+{-| A select dropdown with custom attributes.
```

Comment:
Can we phase this out now? (💀 💀 💀)

Reply:
(or at least maybe remove it from the API?)

Reply:
Yeah! While I'm at it, suppose I might as well get design to take a 👁 to see if there is anything needed for the current design so we don't need to weirdly customize!